src/src/main.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class main {
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        int glückszahl= 7;
```

Comment:
keine sonderzeichen im programmcode - die machen oft probleme in variablennamen (ue stattdessen)